_profiles/drupal/resource-versioning/index.md

```diff
@@ -0,0 +1,301 @@
+---
+name: Resource Versioning
+short_description: |
+  Defines a protocol for requesting versioned resources via a JSON:API
+  server.
+
+extended_description: |
+  # Overview
+  JSON:API servers that are capable of tracking a record of changes to a
+  resource object may afford the capability of requesting a resource object
+  as it existed in a prior or successive state. This capability is herein
+  defined as _resource versioning_.
+  
+  This profile establishes a protocol for resource versioning by defining a
+  query parameter and its semantics in order to identify arbitrary revisions
```

Comment:
revisions → versions :)

Reply:
Not in this instance, remember: "not all revisions are versions, but all versions are revisions"